### ISSUE TITLE
Update support details to point to support relevant to addons.thunderbird.net

### ISSUE THREAD

**Issue by muffinresearch** [thundernest/addons-server#29]
Clicking the "Report a bug" link on https://addons.thunderbird.net/ points to the support page information for add-ons which is more relevant to Firefox clients. This linkshould be changed to provide support info relevant for Thunderbird.

![themes____add-ons_for_thunderbird](https://user-images.githubusercontent.com/1514/44901816-4ee47380-ad00-11e8-9345-158dcad3385b.jpg)

**Comment by eviljeff** [comment]
I already filled a pull request for this #21

**Comment by muffinresearch** [comment]
@sancus The "report a bug" link in the footer still points to `https://developer.mozilla.org/en-US/Add-ons/AMO/Policy/Contact` and we're still getting reports.

<img width="1091" alt="add-ons_for_thunderbird" src="https://user-images.githubusercontent.com/1514/48543937-789e4700-e8ba-11e8-8596-f5fb5a2727ff.png">

I don't think the patch addressed that link.

**Comment by Sancus** [comment]
@muffinresearch you're correct, it looks like that PR only addressed the link on the 404 page. I'll fix this other link tomorrow.
